src/schema/home/context.ts

```diff
@@ -180,9 +181,78 @@ const Context: GraphQLFieldConfig<
       HomePageModuleContextTrendingType,
     ],
   }),
+  deprecationReason: deprecate({ inVersion: 2, preferUsageOf: "v2_context" }),
   resolve: ({ key, params }, _options, context) => {
     return moduleContext[key](context, params)
   },
 }
 
-export default Context
+const HomePageArtworkOfRelatedToFollowedArtistModuleType = new GraphQLObjectType<
+  any,
+  ResolverContext
+>({
+  name: "HomePageArtworkOfRelatedToFollowedArtistModule",
```

Comment:
@zephraph Perhaps this should just be `Artist` and we can add to the description of the `context` field that `Artist` represents a followed artist?

Reply:
I don't have any suggestions. I'm not in love with it, but it also seems good enough.

Reply:
Fair enough 👍